recipes/m2w64-muparser/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "2.2.5" %}
+
+package:
+  name: m2w64-muparser
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/beltoforion/muparser/archive/v{{ version }}.tar.gz 
+  md5: 02dae671aa5ad955fdcbcd3fee313fb7
+  patches:
+    - no-console.patch
+
+build:
+  number: 0
+  skip: true  # [not win]
+
+requirements:
+  build:
+    - m2w64-toolchain
+    - cmake
+
+test:
+  requires:
+    - python {{ os.environ.get('PY_VER', '') + '*' }}
```

Comment:
I guess that is needed only when MSVC is used. @mingwandroid probably knows better.